pkg/cache/tas_flavor_snapshot.go

```diff
@@ -334,3 +345,10 @@ func (s *TASFlavorSnapshot) fillInCounts(requests resources.Requests) {
 		}
 	}
 }
+
+func notFitMessage(fitCount, totalCount int32) string {
+	if fitCount == 0 {
+		return fmt.Sprintf("topology domain(s) don't allow to fit any of %v pods", totalCount)
```

Comment:
I am not sure the reason why my another comment was not submitted in the previous review...

Let me retry to submit here. Could we surface the topology name in this message?

Reply:
It can be done - pushed in the last commit